SPONSORS.md

```diff
@@ -16,6 +16,7 @@ _Be the first!_
 - [Mercedes-Benz Group](https://github.com/mercedes-benz)
 - [Val Town, Inc.](https://opencollective.com/valtown)
 - [Handsontable - JavaScript Data Grid](https://handsontable.com/docs/react-data-grid/?utm_source=Fastify_GH&utm_medium=sponsorship&utm_campaign=library_sponsorship_2024)
+- [Lokalise - A Localization and Translation Software Tool](https://lokalise.com/?utm_source=Fastify_GH&utm_medium=sponsorship)
```

Comment:
We should alphabetize these.

Reply:
will do!

Reply:
FYI It was in order of appearence: first come first of the list https://github.com/fastify/fastify/blob/main/SPONSORS.md#sponsors

I must add a new sponsor, so I will fix it